stable/redis-ha/templates/redis-sentinel-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-sentinel
+spec:
+  replicas: 2
```

Comment:
I'm new to helm, so I don't know about that. In our environment we have five redis clusters that each have different loads.